### DISCUSSION TITLE
Uno Platform apps monetization

### DISCUSSION THREAD

**Discussion by DmitryBorodiy** [unoplatform/uno#10688] [General]
I have question about Uno Platform app monetization...
How do I implement this?

**Comment by jeromelaban** [comment]
Uno does not provide anything specific for monetization on any platform at this time, but you can rely on native platforms APIs to do this, most are already documented for the Xamarin ecosystem.

Which platforms are of interest to you?

**Reply by DmitryBorodiy** [reply]
Android and Windows UWP
